### ISSUE TITLE
Editorial Edits

### ISSUE THREAD

**Issue by TanyaS08** [PoisotLab/ms_network_prediction#151]
#### Q1

Affiliation 2: Montréal, Canada
Affiliation 3: Montréal, Canada

#### Q2

Affiliation 5: Goiâna, Brasil

#### Q3
[2] vol 94 issue 1 
[51] vol 48 issue 7 
[168] vol 10

#### Q4:
[4] 20131000 
[21] e5725 
[132] 308, there is also an author error here Riva GVD should be Dalla Riva GV

#### Q5
[5] Oxford, United Kingdom 
[103] London, United Kingdom

#### Q6 
[16] Preprint (no volume or page numbers) but should list bioRxiv not Ecology as the 'journal'
[25] vol 195, pages 751 - 758
[26] Preprint (no volume or page numbers) but should list bioRxiv not Ecology as the 'journal'
[33] vol 10, pages 356 - 367
[63] Preprint (no volume or page numbers)
[115] vol 23, pages 465 - 473

All citations between 72-118 are one off, and 119 - 231 are 2 off.

## Additional errata

- line 53: extra parentheses around citation [5,6]
- line 55: "biodiversity shortfalls" -> "biodiversity data shortfalls"
- line 85 (left): "and a myriad infectious diseases" -> "myriad **of** infectious diseases"
- line 106-107 (left col.): "discrete" -> "disjoint"
- line 92 (right col.): "opposed [43]; instead" -> "opposed [43]. Instead," --- (There is already a semicolon at the end of this paragraph, and in the one preceding it)
- line 98-99 (right col): "limited in the size and spatial coverage" -> "limited in the size of their spatial coverage"
- line 166 (left): cut "artificial", unnecessary
- line 179 (left): coexisting should be co-existing as we hyphenate co-occurrence
- lines 181 & 183 (left): features vectors -> feature vector (line 181) and feature vectors (line 183)
- line 185 (left): "each layer is independent from the one before it" --- should be "the weights for each layer" i think?
- line 174 (right): "ADAM optimizer" -> we should probably say the learning rate as well?
- line 305: (right) missing concluding em dash, "at the government level [85] paints" -> "at the govenment level [85]---"
- line 333: (right) "and parameters \theta" -> "and parameters \theta (called weights in the contents of neural networks)"
- line 354 (left): "species interactions networks" -> "species interaction networks"
- line 405 (left): "two parts (i)) " -> " two parts: (i) "
- line 385 (right): "or the models predicts no interaction but"-> "or the model predicts no interaction and"
- line 470 (left): "left with S^2" -> "left with (S-1)^2" ---> no point in considering A_ii
- line 473 (right): "most (if not all) network properties" -> "most (if not all) ecological network propeties" --- not necessarily true for other types of networks
- line 485 (right): extra parenthesis around [111]
- line 494 (right) "interaction to mirror" -> "interaction mirrors" --- matches the tense of the next clause
- line 581 (left): "between species the" -> "between species, the"
- line 613 (left): "for the metacommunity" -> "for metacommunity"
- line 642 (right) "time series-based" -> "time-series-based"
- line 672 (right): "Any empirical time series, we" -> "Any empirical time-series we"
- line 697 (right): "or not routinely or at scale" -> "or not routinely, or at scale"
- line 709 (right): "decision." -> "decisions."
- Figure 1 caption: "i.e. a list of known possible interactions" -> "i.e. a list of co-occurences"
- Figure 2: Spatial network prediction box -> "feasible networks acroos space" -> "across space"

## Figure query

All figures seem to have have their resolution reduced and their fonts replaced with Times. The original font is Roboto, which is available for commercial use under the Apache 2.0 licence so long as that is mentioned in the acknowledgements. If not, tell us what fonts we can use

## Additional queries

On the PDF only the ORCiD for TS, DC, GD, and TP are listed however an ID was provided for all authors.
